### ISSUE TITLE
Date values are inconsistent

### ISSUE THREAD

**Issue by buffaybu** [dtjohnson/xlsx-populate#30]
[test.xlsx](https://github.com/dtjohnson/xlsx-populate/files/874408/test.xlsx)

This sheet has two cells of the same date 2017/04/04 20:00:00 GMT+0800.

But after parsed (in browser):

```
sheet.cell('A1').value() // 42829.8333333333
sheet.cell('B1').value() // 42829.833333333336
XlsxPopulate.numberToDate(42829.8333333333) // 2017/04/04 19:59:59
XlsxPopulate.numberToDate(42829.833333333336) // 2017/04/04 20:00:00
```

Sorry I can't reproduce Cell A1. It was input by a user and I just pasted it there.

**Comment by buffaybu** [comment]
Just checked out sheet1.xml, A1's value is `42829.833333333299` and B1's `42829.833333333336` and according to [dateConverter.js](https://github.com/dtjohnson/xlsx-populate/blob/master/lib/dateConverter.js#L38), `milliseconds` will be `1491393599999.997` and `1491393600000`.

It seems that Date constructor uses `parseInt` when it receives a number so A1 is wrongly `1491393599999`. Maybe here we need `Math.round` before passing it to the constructor?

**Comment by dtjohnson** [comment]
Great catch! Thanks for the PR. I merged it and just published v1.5.2 with your fix.
